### PR TITLE
Release/v0.6.1

### PR DESCRIPTION
**Pull request by NimJay**
This PR releases Bank of Anthos `v0.6.1`.

* [Previous release pull-request (v0.6.0)](https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/1425).
* [Previous-to-previous pull-request (v0.5.11)](https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/1297)

## ToDo
- [x] Create draft release notes (https://github.com/GoogleCloudPlatform/bank-of-anthos/releases/edit/untagged-f64fdc4b59e175c8a22c)
- [x] Clean up PR (manifest duplicates, etc)
- [x] Test on a cluster manually (http://34.67.242.86/)
- [x] Deploy on prod (https://bank-of-anthos.xyz/)

### CODE REVIEW

**Review by donmccasland — COMMENTED** [review]
Holy yaml whitespaces, batman.  Did we need to reformat all of that?

File: docs/releasing/make-release.sh [hunk 1]
```diff
@@ -76,7 +76,7 @@ find "${REPO_ROOT}/${RELEASE_DIR}" -name '*.yaml' -exec sed -i -e "s'value: dev'
 rm ${REPO_ROOT}/${RELEASE_DIR}/*-e
 
 # update version in terraform scripts
-sed -i -e "s@sync_branch  = .*@sync_branch  = \"${NEW_VERSION}\"@g" ${REPO_ROOT}/iac/tf-anthos-gke/terraform.tfvars
+sed -i -e "s@sync_branch  = .*@sync_branch  = \"release/${NEW_VERSION}\"@g" ${REPO_ROOT}/iac/tf-anthos-gke/terraform.tfvars
```
Comment (NimJay) [inline docs/releasing/make-release.sh:79]: Good catch, @bourgeoisor!

Comment (bourgeoisor) [reply docs/releasing/make-release.sh:79]: I noticed it on the very first commit that was generated, where the `release/` was dropped in the diff
Manually fixed it + fixed it here in the script

**Review by minherz — COMMENTED** [review]
lgtm although the formatting is confusing. I wasn't able to compare bit-to-bit the major reformatting like in tansaction-history.yml

File: kubernetes-manifests/accounts-db.yaml [hunk 2]
```diff
@@ -65,29 +67,29 @@ spec:
...
-              name: postgresdb
-              subPath: postgres
+      - envFrom:
+        - configMapRef:
+            name: environment-config
+        - configMapRef:
+            name: accounts-db-config
+        - configMapRef:
+            name: demo-data-config
+        image: us-central1-docker.pkg.dev/bank-of-anthos-ci/bank-of-anthos/accounts-db:v0.6.1@sha256:7c4cf161904b4ef869cc8796d7b5bcde1dda6f861898c0cfed6afd4e4affe659
```
[the remark this answers is not in the document]
Comment (bourgeoisor) [reply kubernetes-manifests/accounts-db.yaml:77]: Yes, it's a security / stability thing. It's technically possible to overwrite Docker tags. The digest allows us to verify that not only are we pulling the right tag, but that the hash matches what we expect it to. This helps prevent e.g. accidental overwrites at the source, or MitM attacks.

**Comment by bourgeoisor** [comment]
@minherz the diff is much easier to read with the "hide whitespace" option:

image

**Pull request merged**